index.bs

```diff
@@ -971,6 +972,24 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     consent=] for (i.e., <em>authorizes</em>) a [=ceremony=] to proceed. This MAY involve [=user verification=] if the
     employed [=authenticator=] is capable, or it MAY involve a simple [=test of user presence=].
 
+: <dfn>Backup</dfn>
+: <dfn>Backed Up</dfn>
+:: [=Public Key Credential Sources=] may be [=backed up=] in some fashion such that they may become present on an authenticator other 
```

Comment:
(nit, not blocking: these definitions link to themselves several times. Seems superfluous!)